packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -4253,7 +4253,7 @@ void main() {
       selection,
       equals(
         const TextSelection(
-          baseOffset: 10,
```

Comment:
We were testing the wrong behavior, the refactoring exposes this bug

Reply:
Nice, these were clearly wrong in retrospect.